src/cmd/login/login.go

```diff
@@ -0,0 +1,77 @@
+package login
+
+import (
+	"context"
+	"github.com/otterize/otterize-cli/src/pkg/config"
+	"github.com/otterize/otterize-cli/src/pkg/login/auth_api"
+	"github.com/otterize/otterize-cli/src/pkg/login/server"
+	"github.com/otterize/otterize-cli/src/pkg/orgs"
+	"github.com/otterize/otterize-cli/src/pkg/users"
+	"github.com/otterize/otterize-cli/src/pkg/utils/prints"
+	"github.com/pkg/browser"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const SwitchAccountFlagKey = "switch"
+
+func login(_ *cobra.Command, _ []string) error {
+	ctxTimeout, cancel := context.WithTimeout(context.Background(), config.DefaultTimeout)
```

Comment:
there have already been some changes in this area in the CLI defined in otterize/tools. Please make sure you align the two. (specifically talking about a change in the context used here to solve a frequent timeout issue with the login command,  but the also holds for changes in the orgs & users apis...)

Reply:
also lets discuss how we avoid maintaining two CLIs in the near future, since this is going to be a repeating issue (maybe involve @orishoshan)?

Reply:
synced with tools